analytics/integration_test/src/integration_test.cc

```diff
@@ -102,6 +102,27 @@ TEST_F(FirebaseAnalyticsTest, TestGetAnalyticsInstanceID) {
   EXPECT_FALSE(future.result()->empty());
 }
 
+TEST_F(FirebaseAnalyticsTest, TestGetSessionID) {
+  firebase::Future<int64_t> future;
+
+  // It can take Analytics a moment to initialize on iOS simulator.
+  // So on iOS/tvOS simulator, retry this test if GetSessionId returns an
+  // error.
+#if TARGET_OS_SIMULATOR
+  FLAKY_TEST_SECTION_BEGIN();
+#endif  // TARGET_OS_SIMULATOR
+
+  future = firebase::analytics::GetSessionId();
+  WaitForCompletion(future, "GetSessionId");
+
+#if TARGET_OS_SIMULATOR
+  FLAKY_TEST_SECTION_END();
+#endif  // TARGET_OS_SIMULATOR
+
+  EXPECT_TRUE(future.result() != nullptr);
+  EXPECT_NE(*future.result(), 0L);
```

Comment:
nit: Shouldn't '0L' be '(int64_t)0' as L isn't consistently defined? In this case it's zero, so probably not a problem, but in the stub implementation (below) it's not.

Reply:
OK yeah, I'll add the static cast.